samples/cpp/gstreamer_pipeline.cpp

```diff
@@ -432,6 +434,14 @@ class GStreamerPipeline
 
 int main(int argc, char *argv[])
 {
-    GStreamerPipeline pipe(argc, argv);
-    return pipe.run();
+    try
+    {
+        GStreamerPipeline pipe(argc, argv);
+        return pipe.run();
+    }
+    catch(Exception& e)
```

Comment:
`const` reference should be here - you are not going to modify exception object.
http://en.cppreference.com/w/cpp/language/try_catch

Reply:
@alalek done